lib/models/Todo.js

```diff
@@ -0,0 +1,82 @@
+const pool = require('../utils/pool');
+
+module.exports = class Todo {
+  id;
+  task;
+  description;
+  complete;
+  user_id;
+  created_at;
+  
+  constructor(row) {
+    this.id = row.id;
+    this.task = row.task;
+    this.description = row.description;
+    this.complete = row.complete;
+    this.user_id = row.user_id;
+    this.created_at = row.created_at;
+  }
+
+  static async insert({ task, description, complete, user_id }) {
+    const { rows } = await pool.query(
+      `
+      INSERT INTO todos (task, description, complete, user_id)
+      VALUES ($1, $2, $3, $4)
+      RETURNING *
+      `,
+      [task, description, complete, user_id]
+    );
```

Comment:
In order for the default value to work on the 'complete' row, consider only entering the task, description, and user_id.